src/components/shared/translations.js

```diff
@@ -5,7 +5,9 @@ const de = {
   errorMandatoryFieldsIntro: 'Bitte korrigieren Sie Ihre Eingabe in folgenden Feldern: ',
   errorMandatoryFieldsOutro: '. Erst dann können Sie Ihre Angaben speichern.',
   errorMandatoryFieldsDefault: 'Bitte füllen Sie alle Pflichtfelder korrekt aus. Erst dann können Sie Ihre Angaben speichern.',
-  placeholderAutoSuggest: "Suchbegriff..."
+  placeholderAutoSuggest: "Suchbegriff...",
+  save: "Speichern",
+  abort: "Abbrechen"
```

Comment:
Please let's use the term "cancel" as key here. While the german term "Abbrechen" is the correct translation for both "Abort" and "Cancel", the term "Abort" is much more harsh, indicating the immediate termination of a process that has already begun, while "Cancel" mildly implies to return to the previous state, which i feel is more default here.

Also, i do not like the term "abort" for its other connotations, be them in english or german.